### DISCUSSION TITLE
reading xlsx issue

### DISCUSSION THREAD

**Discussion by spelts** [PHPOffice/PhpSpreadsheet#2349] [Q&A]
I am receiving an xlsx file from a third party.  the file opens in Excel w/o issue and all data is correct.

however, the data isn't correct when I import in PhpSpreadsheet exactly.  

if i open the xlsx file in Excel and save it back out, the import into phpspreadsheet is all correct.

any ideas.  i can provide more info but figured I start with this info first

thanks

**Comment by oleibman** [comment]
It is probably a namespacing issue. There are several fixes in master which are the start of a multi-phase project to handle namespacing. If you can test against master, please try. Note that your data should be correct, but some items (e.g. styles) were not part of the first phase.

**Reply by spelts** [reply]
thanks, all this github is very new to me so not sure how to find the fixes.

but a little more on issue
I notice when a column has blanks in it, the next column, will typically be read wrong and shift everything

thanks
